README.md

```diff
@@ -1,79 +1,56 @@
-# image-exporter
+# Plotly Image Exporter
 
-aka image server 2.0.
+## Install
 
-See https://github.com/plotly/streambed/issues/9655 for info on the requirements.
```

Comment:
IMO, this should say `npm install -g` (the downstream examples are assuming this)

Reply:
Related to what was discussed in https://github.com/plotly/image-exporter/pull/73#issuecomment-381774163, the downstream examples are assuming that use of the standalone app.